apps/images/factories.py

```diff
@@ -28,9 +28,9 @@ class Meta:
     image = ImageField(from_func=generate_temporary_image)
     width = fuzzy.FuzzyInteger(500, 1000)
     height = fuzzy.FuzzyInteger(500, 1000)
-    title = factory.Faker('sentence', nb_words=10)
+    title = factory.Faker('sentence', nb_words=3)
```

Comment:
It was making the tests flaky because sometimes 10 words length would be larger than allowed max length.

https://github.com/ModularHistory/modularhistory/pull/878/commits/7ac0eee8d5759c5ac84f789e87dac8a805c6559f